tests/repl/test_commandline.rs

```diff
@@ -140,3 +140,18 @@ fn commandline_test_cursor_end() -> TestResult {
 fn commandline_test_cursor_type() -> TestResult {
     run_test("commandline get-cursor | describe", "int")
 }
+
+#[test]
+fn commandline_test_accept() -> TestResult {
+    // Explanation: run_test captures a command's output.
+    // But `--accept` executes a new command that then outputs "accepted"
+    run_test("commandline edit --accept \"print accepted\"", "")
```

Comment:
Shouldn't the expected value contain `accepted`?

Reply:
As stated in the comment above the code: `commandline edit` runs successfully without any output. Instead, it creates a new command that runs `print accepted`. Therefore, the command's output is empty and the successor's output will be "accepted".

Reply:
I can understand this but when I saw it, it seemed a bit weird that my history would have "commandline edit --accept "print accepted"" and then another entry for "print accepted". I mean, I understand it, but it was unexpected.